src/main/java/core/basesyntax/service/impl/FileWrite.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service.impl;
+
+public interface FileWrite {
+    void write(String result, String nameFileWrite);
```

Comment:
According to the checklist, it's recommended to avoid hardcoding values. The method signature for `write` should take a `filePath` as a parameter instead of `nameFileWrite` to make it more flexible and clear.